ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/policy-maps.md

```diff
@@ -108,4 +113,13 @@
                 # Range in bytes <256-128000000>.
                 higher_rate_burst_size: <int>
                 higher_rate_burst_size_unit: <str; "bytes" | "kbytes" | "mbytes" | "packets"; default="bytes">
+      copp:
+
+        # Name of control-plane poilcy.
+        name: <str>
```

Comment:
After verifying on several platforms, we do not need to implement a custom name. Instead we should add a dict level for `copp_system_policy`, since this is a hardcoded name.

Reply:
alternatively we can use `valid_values` and only accept this name. @gmuloc @carlbuchmann ?

Reply:
please just implement the dict `copp_system_policy` instead of `copp` since that is the only supported value. We can revisit later if needed.

Reply:
Fixed.